Docs/source/acknowledge.rst

```diff
@@ -0,0 +1,24 @@
+Acknowledge WarpX
+=================
+
+Please acknowledge the role that WarpX played in your research by adding the following sentence to your publications, it helps contributors keep track of users and promote the project:
```

Comment:
Thanks for this PR @MaxThevenet !
It is very useful to have a prepared sentence.
But I would not phrase it like that "contributors (want to) keep track of users". Maybe more "community" centered would be nicer.

Reply:
Oh yes, thanks, I'll improve that!